packages/terra-time-input/tests/wdio/time-input-twelve-hour-spec.js

```diff
@@ -124,70 +64,6 @@ Terra.describeViewports('Time Input Twelve Hour', ['medium'], () => {
-      browser.refresh();
-      Terra.hideInputCaret('#timeInput input[name="terra-time-minute-time-input"]');
-
-      $('#timeInput input[name="terra-time-hour-time-input"]').click();
-      browser.keys('12');
-      browser.keys('00');
-      $('#timeInput input[name="terra-time-minute-time-input"]').click();
-      browser.keys('-');
-      expect($('#timeInput input[name="terra-time-hour-time-input"]')).toHaveValue('11');
-      expect($('#timeInput input[name="terra-time-minute-time-input"]')).toHaveValue('59');
-      validateElement('- causes rollover to morning');
-    });
-
-    it('should roll over to 11 PM if hour and minute is at 12:00 AM', () => {
-      browser.url('/raw/tests/cerner-terra-framework-docs/time-input/twelve-hour/filled-morning');
-      browser.refresh();
-      Terra.hideInputCaret('#timeInput input[name="terra-time-minute-time-input"]');
-
-      $('#timeInput input[name="terra-time-hour-time-input"]').click();
-      browser.keys('12');
-      browser.keys('00');
-      $('#timeInput input[name="terra-time-minute-time-input"]').click();
-      browser.keys('-');
-      expect($('#timeInput input[name="terra-time-hour-time-input"]')).toHaveValue('11');
-      expect($('#timeInput input[name="terra-time-minute-time-input"]')).toHaveValue('59');
-      validateElement('- causes rollover to evening');
-    });
-  });
-
-  describe('pressing +', () => {
-    it('should roll over to 12 AM if hour and minute is at 11:59 PM', () => {
-      browser.url('/raw/tests/cerner-terra-framework-docs/time-input/twelve-hour/filled-evening');
-      browser.refresh();
-      Terra.hideInputCaret('#timeInput input[name="terra-time-minute-time-input"]');
-
-      $('#timeInput input[name="terra-time-hour-time-input"]').click();
-      browser.keys('11');
-      browser.keys('59');
-      $('#timeInput input[name="terra-time-minute-time-input"]').click();
-      browser.keys('+');
-      expect($('#timeInput input[name="terra-time-hour-time-input"]')).toHaveValue('12');
-      expect($('#timeInput input[name="terra-time-minute-time-input"]')).toHaveValue('00');
-      validateElement('+ causes rollover to morning');
-    });
-
-    it('should roll over to 12 PM if hour and minute is at 11:59 AM', () => {
-      browser.url('/raw/tests/cerner-terra-framework-docs/time-input/twelve-hour/filled-morning');
-      browser.refresh();
-      Terra.hideInputCaret('#timeInput input[name="terra-time-minute-time-input"]');
-
-      $('#timeInput input[name="terra-time-hour-time-input"]').click();
-      browser.keys('11');
-      browser.keys('59');
-      $('#timeInput input[name="terra-time-minute-time-input"]').click();
-      browser.keys('+');
-      expect($('#timeInput input[name="terra-time-hour-time-input"]')).toHaveValue('12');
-      expect($('#timeInput input[name="terra-time-minute-time-input"]')).toHaveValue('00');
-      validateElement('+ causes rollover to evening');
-    });
-  });
-
```

Comment:
Why are these tests removed? Instead the click events before pressing +/- should be updated so that they match the expected output.

Reply:
Already we have added +/- hotkey test cases in time-input-spec file.

Reply:
Updated